src/hyperloglog.c

```diff
@@ -227,6 +231,19 @@ static int simd_enabled = 1;
 #define HLL_USE_AVX2 0
 #endif
 
+#ifdef __ARM_NEON
+static int simd_enabled = 1;
+#define HLL_USE_NEON (simd_enabled)
+#else
+#define HLL_USE_NEON 0
+#endif
+
```

Comment:
Do we need add a runtime check here when server startup ?

Reply:
I've limited this to aarch64, which is guaranteed to have neon.

Reply:
It is guaranteed to have NEON support in the compile environment. Is it possible to run AArch64 binaries on a non-neon platform?

Reply:
Yes, I'm wondering if we compiled on the AArch64 platform and released the ARM version binary, but the binary is running on an older ARM platform that doesn't support the NEON architecture (e.g., ARMv6). If this edge case isn't a concern, I'm fine with skipping the runtime checker.

Reply:
> Yes, I'm wondering if we compiled on the AArch64 platform and released the ARM version binary, but the binary is running on an older ARM platform that doesn't support the NEON architecture (e.g., ARMv6). If this edge case isn't a concern, I'm fine with skipping the runtime checker.

https://developer.arm.com/documentation/102474/0100/Fundamentals-of-Armv8-Neon-technology
```
AArch64 is the name used to describe the 64-bit Execution state of the Armv8-A architecture. In AArch64 state, the processor executes the A64 instruction set, which contains Neon instructions (also referred to as SIMD instructions). GNU and Linux documentation sometimes refers to AArch64 as ARM64.
```